M/MagicVLSI/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MagicVLSI"
+version = v"8.3.61"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Keno/magic.git", "677b176b2fdf8b623e81b8b09b598bdf56d3c130")
```

Comment:
Wrong git hash? https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=6118&view=logs&j=49bf9737-d23a-5997-c992-3db4d9b68d6d&t=3668ab0d-2e18-5e0b-5ad0-c71e20c82665

Reply:
Maybe I rebased after I pushed this.
```suggestion
    GitSource("https://github.com/Keno/magic.git", "a58100a14cdb57462cf75691532d01c5733fb59b")
```